build.gradle.kts

```diff
@@ -62,9 +62,11 @@ spotless {
     target("**/*.kt")
     ktfmt().googleStyle()
   }
-  format("swift") {
-    target("iosApp/iosApp/**/*.swift")
-    nativeCmd("swiftFormat", "/usr/bin/env", listOf("swift", "format"))
+  if (System.getProperty("os.name").contains("Mac OS X")) {
+    format("swift") {
+      target("iosApp/iosApp/**/*.swift")
+      nativeCmd("swiftFormat", "/usr/bin/env", listOf("swift", "format"))
+    }
```

Comment:
unrelated: the swift formatter isn't working for me on linux (even with swift installed) so I'm gating it with an OS check